### DISCUSSION TITLE
does not load hyperswitch-web open source payment methods

### DISCUSSION THREAD

**Discussion by jkazuo55** [juspay/hyperswitch#3788] [General]
i am setting everything up by default 

in my hyperswitch-control-center i configured
    apiBaseUrl=http://localhost:8080
    sdkBaseUrl=http://localhost:9050

in my hyperswitch-web
  STATIC_DIR=./dist
  HYPERSWITCH_PUBLISHABLE_KEY="pk_snd_f51a0e3f2******" 
  HYPERSWITCH_SECRET_KEY="snd_NJ3TlciRiJXzhTvonSHEyf*********"
  HYPERSWITCH_SERVER_URL="http://localhost:8080"
  HYPERSWITCH_CLIENT_URL="http://localhost:9000"
  SELF_SERVER_URL="http://localhost:5252"

When trying to open http://localhost:9060/ i get the following error in the browser 
   in console 
      
![image](https://github.com/juspay/hyperswitch/assets/15249550/c014010d-3dcf-42c2-b3d3-20c4ec9505ce)

  in network
  
![image](https://github.com/juspay/hyperswitch/assets/15249550/c27736ae-44b2-49ed-aeb4-412662cab2c4)

helpme please! :(

**Comment by ArushKapoorJuspay** [comment]
Hey [jkazuo55](https://github.com/jkazuo55),

the base port where HyperLoader.js file runs is localhost:9050 which is why you are getting the 404 Not Found. You can solve this issue using one of two ways :-

1. Change the HYPERSWITCH_CLIENT_URL value to http://localhost:9050/

2. In hyperswitch-web, you will have to make the following changes -
    A. Inside `webpack.dev.js`, update the port from `9050` to your desired port (in your case 9000)

<img width="634" alt="Screenshot 2024-02-26 at 12 26 42 PM" src="https://github.com/juspay/hyperswitch/assets/121166031/4459e3e4-31a6-416c-b819-c410f3defc42">

B. Inside `webpack.common.js` update the `sdkUrl` value to the same localhost port which you configured in the previous step (in 
        your case http://localhost:9000)

<img width="493" alt="Screenshot 2024-02-26 at 12 27 05 PM" src="https://github.com/juspay/hyperswitch/assets/121166031/76dac983-3f19-41fc-9a11-f5f954f97e03">

**Reply by SanchithHegde** [reply]
@jkazuo55 Just checking in, were you able to set it up?

**Comment by SanchithHegde** [comment]
The author of the discussion has not responded further, will be closing this as resolved in the interest of reducing the number of stale discussions.
